src/assets/i18n/en.json5

```diff
@@ -5993,13 +5991,325 @@
+  "admin.registries.services-formats.modify.failure.head": "Failed Edit",
+  "admin.registries.services-formats.modify.failure.content": "The service has not been edited",
+  "ldn-service-notification.created.success.title": "Successful Create",
+  "ldn-service-notification.created.success.body": "The service has been created",
+  "ldn-service-notification.created.failure.title": "Failed Create",
+  "ldn-service-notification.created.failure.body": "The service has not been created",
+  "ldn-service-notification.created.warning.title": "Please select at least one Inbound Pattern",
+  "ldn-enable-service.notification.success.title": "Successful status updated",
+  "ldn-enable-service.notification.success.content": "The service status has been updated",
+  "ldn-service-delete.notification.success.title": "Successful Deletion",
+  "ldn-service-delete.notification.success.content": "The service has been deleted",
+  "ldn-service-delete.notification.error.title": "Failed Deletion",
+  "ldn-service-delete.notification.error.content": "The service has not been deleted",
+  "service.overview.reset-form.reset-return": "Cancel",
+  "service.overview.delete": "Delete service",
+  "ldn-edit-service.title": "Edit service",
+  "ldn-edit-service.form.label.name": "Name",
+  "ldn-edit-service.form.label.description": "Description",
+  "ldn-edit-service.form.label.url": "Service URL",
+  "ldn-edit-service.form.label.ldnUrl": "LDN Inbox URL",
+  "ldn-edit-service.form.label.inboundPattern": "Inbound Pattern",
+  "ldn-edit-service.form.label.noInboundPatternSelected": "No Inbound Pattern",
+  "ldn-edit-service.form.label.selectedItemFilter": "Selected Item Filter",
+  "ldn-edit-service.form.label.selectItemFilter": "No Item Filter",
+  "ldn-edit-service.form.label.automatic": "Automatic",
+  "ldn-edit-service.form.label.addInboundPattern": "+ Add more",
+  "ldn-edit-service.form.label.submit": "Save",
+  "ldn-edit-service.breadcrumbs": "Edit Service",
+  "ldn-service.control-constaint-select-none": "Select none",
+
+  "ldn-register-new-service.notification.error.title": "Error",
+  "ldn-register-new-service.notification.error.content": "An error occurred while creating this process",
+  "ldn-register-new-service.notification.success.title": "Success",
+  "ldn-register-new-service.notification.success.content": "The process was successfully created",
+
+  "info.coar-notify-support.title": "Notify Support",
+
+  "info.coar-notify.breadcrumbs": "Notify Support",
+
+  "submission.sections.notify.info": "The selected service is compatible with the item according to its current status. {{ service.name }}: {{ service.description }}",
+
+  "item.qa-event-notification.check.notification-info": "There are {{num}} pending review to check",
+
+  "item.qa-event-notification-info.check.button": "Check",
+
+  "mydspace.qa-event-notification.check.notification-info": "There are {{num}} pending review to check",
+
+  "mydspace.qa-event-notification-info.check.button": "Check",
```

Comment:
There are 4 keys in a row here that are duplicates.  The keys on 6165-6171 can be removed because they already appear in this file.